### ISSUE TITLE
How to handle Multi-Status 207 Http Response in Refit

### ISSUE THREAD

**Issue by suyogkhachane** [reactiveui/refit#1335]
Hi,
I'm trying to handle Multi-Status 207 response from one of API.

I have created interface like:
`public interface IServiceAPIs
 {
[Post("/api/data/generate")]
Task GenerateData([Body]generateDataRequest)
}`

and I'm receiving 200 Ok as well as 207 Multi-Status with content if some generation fails.
If StatusCode other than 200 range Refit generates ApiException but with 207 it is directly returning as I'm using Task

How can I handle this.?
Thanks in advance.

**Comment by anaisbetts** [comment]
I don't think that Refit is a good fit for this use-case, sorry :(  I recommend adding a handler for this particular method as an extension method to your interface and write the handler by-hand
